### PR TITLE
refactor for EventCardData in props for EventCards

### PR DESCRIPTION
**Pull request by Fattimo**
refactor with type eventCardData. moved about to be apart of type Eve…ntBase so that there is body text for the eventCard to access

### CODE REVIEW

File: components/events/EventsPageEventCard.tsx [hunk 1]
```diff
@@ -92,16 +82,22 @@ const EventsPageEventCard: React.FC<Props> = ({
...
+                eventCardData.duration /*TODO: replace with formatting from figma*/
+              }
             </CoreTypography>
             <CoreTypography variant="h4" className={header}>
-              {headerText}
+              {eventCardData.name}
             </CoreTypography>
             <CoreTypography className={clsx(body, truncate)}>
-              {bodyText}
+              {eventCardData.about}
```
Comment (PC98) [inline components/events/EventsPageEventCard.tsx:100]: actually we want this to be the nonprofit name, so `eventCardData.nonprofitId.name`

File: server/actions/events.ts [hunk 2]
```diff
@@ -19,7 +19,8 @@ const CARD_FIELDS: Record<keyof EventCardDataType, 1> = {
   image: 1,
   address: 1,
   nonprofitId: 1,
-  duration: 1
+  duration: 1,
+  about: 1
```
Comment (PC98) [inline server/actions/events.ts:23]: you should undo this change, and the changes to `EventBase` and `Event` types

File: components/events/EventsPageEventCard.tsx [hunk 3]
```diff
@@ -92,16 +82,22 @@ const EventsPageEventCard: React.FC<Props> = ({
...
+                eventCardData.duration /*TODO: replace with formatting from figma*/
+              }
             </CoreTypography>
             <CoreTypography variant="h4" className={header}>
-              {headerText}
+              {eventCardData.name}
             </CoreTypography>
             <CoreTypography className={clsx(body, truncate)}>
-              {bodyText}
+              {eventCardData.name}
```
Comment (PC98) [inline components/events/EventsPageEventCard.tsx:100]: Should be `eventCardData.nonprofitId.name` 🙂

Comment (Fattimo) [reply components/events/EventsPageEventCard.tsx]: Wow that was really bad from me, I didn't read the original message closely enough 😅

Comment (PC98) [reply components/events/EventsPageEventCard.tsx]: np!

**Review by PC98 — APPROVED** [review]
